README.md

```diff
@@ -144,7 +144,7 @@ GraphQL-AnyCable uses [anyway_config] to configure itself. There are several pos
     GRAPHQL_ANYCABLE_USE_CLIENT_PROVIDED_UNIQ_ID=false
     ```
 
- 2. YAML configuration files:
+ 2. YAML configuration files (this is **config/graphql_anycable.yml**, *not* **config/anycable.yml**):
```

Comment:
Purely aesthetical suggestion:

```suggestion
 2. YAML configuration files (note that this is `config/graphql_anycable.yml`, *not* `config/anycable.yml`):
```

Reply:
fine with me. i usually am pedantic and only use code blocks for things that are code, but i agree it looks nicer. and also, what *is* code, man?